docs/architecture/porting-existing-aspnet-apps/architectural-differences.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Architectural differences between ASP.NET MVC and ASP.NET Core
+description: Although at a high level ASP.NET MVC and ASP.NET Core share many concepts, there are many differences between them architecturally. Knowing these differences will help teams make informed decisions about how the will migrate their applications to .NET Core.
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Architectural differences between ASP.NET MVC and ASP.NET Core
+
+[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+
+*Notes*
+This will be a high level comparison between the architectures of the two different types of apps which can link to existing documentation for more details (as opposed to Chapter 4, which is an example of applying the changes to eShop).
+
+Topics:
+
+- Startup
```

Comment:
Should logging be added to the topics list?

Reply:
+1 for logging